lib/workspace.rb

```diff
@@ -0,0 +1,47 @@
+require "dotenv"
+require "httparty"
+require_relative "user"
+require_relative "channel"
+Dotenv.load
+
+class Workspace
+  attr_reader :user, :channel
+  attr_accessor :selected
+
+  class SlackApiError < StandardError; end
+
+  def select_channel(selected)
+    selected
```

Comment:
What is the purpose of this method?  It takes an argument and just returns the argument?